src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/NatureOfControlTypeMapping.java

```diff
@@ -62,52 +92,103 @@ public class NatureOfControlTypeMapping {
+        "OE_VOTINGRIGHTS_MORETHAN25PERCENT_AS_CONTROLOVERTRUST");
+    OVERSEAS_ENTITIES_CONTROL_OVER_TRUST_MAP.put(APPOINT_OR_REMOVE_MAJORITY_BOARD_DIRECTORS,
+        "OE_RIGHTTOAPPOINTANDREMOVEDIRECTORS_AS_CONTROLOVERTRUST");
+    OVERSEAS_ENTITIES_CONTROL_OVER_TRUST_MAP.put(SIGNIFICANT_INFLUENCE_OR_CONTROL,
+        "OE_SIGINFLUENCECONTROL_AS_CONTROLOVERTRUST");
+
+    OVERSEAS_ENTITIES_PERSON_JURISDICTIONS_MAP = new EnumMap<>(NatureOfControlJurisdictionType.class);
+    OVERSEAS_ENTITIES_PERSON_JURISDICTIONS_MAP.put(ENGLAND_AND_WALES, "OE_REGOWNER_AS_NOMINEEPERSON_ENGLANDWALES");
+    OVERSEAS_ENTITIES_PERSON_JURISDICTIONS_MAP.put(SCOTLAND, "OE_REGOWNER_AS_NOMINEEPERSON_SCOTLAND");
+    OVERSEAS_ENTITIES_PERSON_JURISDICTIONS_MAP.put(NORTHERN_IRELAND, "OE_REGOWNER_AS_NOMINEEPERSON_NORTHERNIRELAND");
+
+    OVERSEAS_ENTITIES_OTHER_ENTITY_JURISDICTIONS_MAP = new EnumMap<>(NatureOfControlJurisdictionType.class);
+    OVERSEAS_ENTITIES_OTHER_ENTITY_JURISDICTIONS_MAP.put(ENGLAND_AND_WALES, "OE_REGOWNER_AS_NOMINEEANOTHERENTITY_ENGLANDWALES");
+    OVERSEAS_ENTITIES_OTHER_ENTITY_JURISDICTIONS_MAP.put(SCOTLAND, "OE_REGOWNER_AS_NOMINEEANOTHERENTITY_SCOTLAND");
+    OVERSEAS_ENTITIES_OTHER_ENTITY_JURISDICTIONS_MAP.put(NORTHERN_IRELAND, "OE_REGOWNER_AS_NOMINEEANOTHERENTITY_NORTHERNIRELAND");
+
+    OVERSEAS_ENTITIES_CONTROL_OVER_FIRM_MAP = new EnumMap<>(NatureOfControlType.class);
+    OVERSEAS_ENTITIES_CONTROL_OVER_FIRM_MAP.put(OVER_25_PERCENT_OF_SHARES,
+        "OE_OWNERSHIPOFSHARES_MORETHAN25PERCENT_AS_CONTROLOVERFIRM");
+    OVERSEAS_ENTITIES_CONTROL_OVER_FIRM_MAP.put(OVER_25_PERCENT_OF_VOTING_RIGHTS,
+        "OE_VOTINGRIGHTS_MORETHAN25PERCENT_AS_CONTROLOVERFIRM");
+    OVERSEAS_ENTITIES_CONTROL_OVER_FIRM_MAP.put(APPOINT_OR_REMOVE_MAJORITY_BOARD_DIRECTORS,
+        "OE_RIGHTTOAPPOINTANDREMOVEDIRECTORS_AS_CONTROLOVERFIRM");
+    OVERSEAS_ENTITIES_CONTROL_OVER_FIRM_MAP.put(SIGNIFICANT_INFLUENCE_OR_CONTROL,
+        "OE_SIGINFLUENCECONTROL_AS_CONTROLOVERFIRM");
   }
 
   private NatureOfControlTypeMapping() {
     throw new IllegalAccessError("Use the static method designation");
   }
 
-  /**
-   * Collects the nature of control types from the dto and maps them to the api values
-   *
-   * @param personNatureOfControlTypes
-   * @param trusteesNatureOfControlTypes
-   * @param firmNatureOfControlTypes
-   * @return a list of nature of control types
-   */
+    /**
+     * Collects the nature of control types from the dto and maps them to the api values
+     * @param naturesOfControlCollection
+     * @return
+     */
   public static List<String> collectAllNatureOfControlsIntoSingleList(
-      List<NatureOfControlType> personNatureOfControlTypes,
-      List<NatureOfControlType> trusteesNatureOfControlTypes,
-      List<NatureOfControlType> firmNatureOfControlTypes
-  ) {
+          NaturesOfControlCollectionBuilder.NaturesOfControlCollection naturesOfControlCollection) {
```

Comment:
Formatting looks to be out here, Needs to be indented 4 spaces.

Reply:
Changed to use 4 spaces.